rfcs/test_scoring_change.md

```diff
@@ -0,0 +1,70 @@
+# RFC: Change results aggregation on wpt.fyi and visualization on Interop-20** views
+
+## Summary
+
+The aggregate scoring that is displayed in the results view of wpt.fyi is
+inaccurate in some aspects. This proposal will describe a change to the way
```

Comment:
Can we also remove the word "accurate" here? Maybe "suboptimal" or something instead.